lib/manageiq/active_record_connector.rb

```diff
@@ -41,7 +41,8 @@ def self.establish_connection_if_needed(db_config, log_path = nil)
 
       unless existing_connection
         ActiveRecord::Base.logger ||= get_logger_from log_path
-        ActiveRecord::Base.establish_connection(db_config)
+        ActiveRecord::Base.configurations = connection_configurations_from db_config
+        ActiveRecord::Base.establish_connection
```

Comment:
@NickLaMuro I think I follow this change...

LOL At that long commit message "Favor DATABASE_URL in ActiveRecordConnector".  Basically, you're saying we need to set the configuration but not pass it to `establish_connection` so it can decide to use an existing `DATABASE_URL` or the configuration we just set.  AMIRITE?